builder/builder.go

```diff
@@ -78,7 +108,7 @@ func (b *Builder) Build() error {
 	carto := mss.New()
 
 	for _, mss := range b.mss {
-		err := carto.ParseFile(mss)
```

Comment:
I'm not sure I fully understand. Can you expand? Internally ParseFile reads all file contents into a buffer and calls ParseString.